app/src/main/res/layout/activity_sign_up.xml

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@drawable/back_sign_up">
+
+    <com.google.android.material.textfield.TextInputLayout
```

Comment:
Please use styles here to manage the duplicated resources and configurations that was used here.